packages/better-php-doc-parser/src/PhpDocNodeFactory/MultiPhpDocNodeFactory.php

```diff
@@ -61,16 +61,13 @@ public function getTagValueNodeClassesToAnnotationClasses(): array
             IdTagValueNode::class => 'Doctrine\ORM\Mapping\Id',
             GeneratedValueTagValueNode::class => 'Doctrine\ORM\Mapping\GeneratedValue',
             JoinColumnTagValueNode::class => 'Doctrine\ORM\Mapping\JoinColumn',
-
```

Comment:
@TomasVotruba removing one empty line also doesn't make sense :) It makes sense to replace multiple empty lines by one. Sometimes it is handy to group lines somehow.

Reply:
What ECS rule is causing this?

Reply:
I didn't notice it. We will find it later